scripts/makepeds_psana

```diff
@@ -135,6 +133,45 @@ xtcav_lasOff()
    exit
 }
 
+# Arg: DETECTOR class.  This could be a comma-separated list, but we're 
+# matching it in the config file as a string.
+get_config()
+{
+    awk 'BEGIN{n=0;}/'$1' \{/{n=1;next;}/\}/{n=0;next;}{if(n)print;}' <$CFGFILE >/tmp/script.$$
+    source /tmp/script.$$
+    rm /tmp/script.$$
+    #overwrite if argument is passed
+    if [ $MYNOISESIGMIN -ne -1 ]; then
+	NOISESIGMIN=$MYNOISESIGMIN
+    fi
+    if [ $MYNOISESIGMAX -ne -1 ]; then
+	NOISESIGMAX=$MYNOISESIGMAX
+    fi
+    if [ $MYADUMIN -ne -1 ]; then
+	ADUMIN=$MYADUMIN
+    fi
+    if [ $MYADUMAX -ne -1 ]; then
+	ADUMAX=$MYADUMAX
+    fi
+}
+
+# Args: Detector ID, Additional command to run.
+deploy_geometry()
+{
+    GEOPATH=`calibfile path -e $EXP -s $1 -t geometry | awk '{print $2;}' -`
+    GEODIR=`dirname $GEOPATH`
+    if [ ! -d $GEODIR ]; then
+	echo No geometry file found, deploying.
+	geometry_deploy_constants -e $EXP -r $RUN -d $1 -D -c ./calib
```

Comment:
don't hard code -c (which is also wrong as written), default to standard calib-dir. Could add optional directory later.
```suggestion
	geometry_deploy_constants -e $EXP -r $RUN -d $1 -D
```